scripts/automatic.ps1

```diff
@@ -27,23 +26,35 @@ if (!(Test-Path $automatic_runtime_unpacked)) {
     Expand-Archive $automatic_runtime_package -DestinationPath $automatic_runtime_unpacked
 }
 
-$plugins_package_dir = $automatic_package_dir + "\plugins"
-$automatic_runtime_dir = $plugins_package_dir + "\ya-automatic-ai"
-$automatic_runtime_dir_subdir = $automatic_runtime_dir + "\automatic"
+$plugins_package_dir = "$($automatic_package_dir)\plugins"
+$automatic_runtime_dir = "$($plugins_package_dir)\ya-automatic-ai"
+
+if (Test-Path $automatic_runtime_dir) {
+    Remove-Item -Path $automatic_runtime_dir -Force -Recurse
+}
+
+$automatic_runtime_dir_subdir = "$($automatic_runtime_dir)\automatic"
 New-item $automatic_runtime_dir_subdir -ItemType Directory -Force
 
-$automatic_runtime_unpacked_subdir_pattern = $automatic_runtime_unpacked + "\" + $automatic_runtime_package_subdir + "\*"
+$automatic_runtime_unpacked_subdir_pattern = "$($automatic_runtime_unpacked)\$($automatic_runtime_package_subdir)\*"
 Copy-Item -Path $automatic_runtime_unpacked_subdir_pattern -Destination $automatic_runtime_dir_subdir -Recurse
-$automatic_runtime_dir_dst = $automatic_runtime_dir + "\"
+$automatic_runtime_dir_dst = "$($automatic_runtime_dir)\"
 Copy-Item "scripts\config.json" -Destination $automatic_runtime_dir_dst
-$plugins_package_dir_dst = $plugins_package_dir
-Copy-Item "scripts\ya-automatic-ai.json" -Destination $plugins_package_dir_dst
+
+
+$automatic_runtime_package_descriptor = "$($plugins_package_dir)\ya-automatic-ai.json"
+if (Test-Path $automatic_runtime_package_descriptor) {
+    Remove-Item -Path $automatic_runtime_package_descriptor -Force -Recurse
+}
+Copy-Item "scripts\ya-automatic-ai.json" -Destination $automatic_runtime_package_descriptor
 
 if ($compress) {
-    $automatic_package_dir_pattern = $automatic_package_dir + "\*"
-    $automatic_dist_package = $bin_dir + "\dist_package.zip"
+    $workspace = $(Get-Location)
+    $automatic_dist_package = "$($workspace)\$($bin_dir)\dist_package.zip"
     if (Test-Path $automatic_dist_package) {
-        Remove-Item $automatic_dist_package
+        Remove-Item $automatic_dist_package -Force -Recurse
     }
-    Compress-Archive -Path $automatic_package_dir_pattern -DestinationPath $automatic_dist_package -CompressionLevel $compression_lvl
+    Set-Location $automatic_package_dir
+    tar.exe -a -cf $automatic_dist_package .\*
```

Comment:
GH app was incapable of uncompressing archive created using `compress-archive` 
(could be because of https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.archive/compress-archive?view=powershell-7.4#notes or because of the way it handles archives over 2GB)